packages/pino/rollup.config.cjs.js

```diff
@@ -0,0 +1,16 @@
+import typescript from '@rollup/plugin-typescript';
```

Comment:
HI just chiming in :) 

You can actually combine these two up into 1 file no need to necesarily split them. As it accepts an array, you can put both in the same array in 1 file. That simplifies needing to do `build cjs && build esm` everywhere etc.

Reply:
hi, I've tried that but it was throwing errors, I can't remember what was it, but I will give it another shot.

Reply:
so mostly the problem is with `outDir`:

```
 [!] (plugin typescript) RollupError: @rollup/plugin-typescript: Path of Typescript compiler option 'outDir' must be located inside Rollup 'dir' option.
 ```

When I try to define the plugin for each one separately, the setup goes sideways:

```ts
export default [
  {
    input: 'src/index.ts',
    output: [
      {
        dir: 'dist/esm',
        format: 'esm',
        exports: 'named',
        sourcemap: true,
        preserveModules: true,
        plugins: [typescript({ outDir: 'dist/esm', declarationDir: 'dist/esm/types' })],
      },
      {
        dir: 'dist/cjs',
        format: 'cjs',
        exports: 'named',
        sourcemap: true,
        preserveModules: true,
        entryFileNames: '[name].cjs',
        plugins: [typescript({ outDir: 'dist/cjs', declarationDir: 'dist/cjs/types', module: "commonjs" })],
      },
    ],
    plugins: [
      replace({
        preventAssignment: true,
        AXIOM_VERSION: process.env.npm_package_version,
      }),
    ],
  },
];
```
resulting in error:

```
@axiomhq/js:build: src/index.ts → dist/esm, dist/cjs...
@axiomhq/js:build: [!] RollupError: Could not resolve "./client.js" from "src/index.ts"
```